### ISSUE TITLE
Request for Filter Option

### ISSUE THREAD

**Issue by harshitSingh1** [emmabostian/developer-portfolios#1615]
Its very hard to look into all the portfolios. Please add filter option so that people can easily filter out portfolios based on profession like "Software Developer", "Content Writer", "Full stack developer", "Marketing Analyst", "SEO Analyst" etc...

**Comment by replyre** [comment]
@harshitSingh1  @emmabostian, For this issue, I have an idea: since there is still no way to add a JS script in Readme, what we can do is to create a simple webpage, scrape and add all the data in a JSON file, and add whatever filters we want and host it on this site.

**Comment by harshitSingh1** [comment]
Yes, you can either create a webpage or add tags in the README file.
